### ISSUE TITLE
iPhone 6s+ Stuck in Reboot Loop After Disabling Dark Mode

### ISSUE THREAD

**Issue by maxsxm** [checkra1n/BugTracker#126]
Everything was working fine after jailbreaking my iPhone 6s+ (iOS 13.3 Beta 1) using checkra1n, until I accidentally turned off Dark Mode, which froze the phone for 5 minutes before respringing 3 times, then entering a boot loop. Only restoring it works.

**Comment by littlelailo** [comment]
Did you install any tweaks? And did you try to boot jailbroken again?

**Comment by maxsxm** [comment]
> Did you install any tweaks? And did you try to boot jailbroken again?

Yes, I installed some minor tweaks. I havent tried to reboot the jailbreak yet.

**Comment by littlelailo** [comment]
Try to boot with safe mode enabled

**Comment by littlelailo** [comment]
Do you still have the issues with the newest version/were you able to resolve it using safe mode?
